src/components/Layout/index.jsx

```diff
@@ -111,24 +116,23 @@ class Layout extends Component {
         {isNormal
           && !isHome && <TitleBanner title={title} hasSearch={hasSearch} />}
 
-        {isCenterPage && (
-          <div className={styles.pageTitle}>
-            <div className={styles.title}>
-              <Icon
-                onClick={this.goBack}
-                name="previous"
-                size={20}
-                type="dark"
-                className={styles.icon}
-              />
-              {pageTitle}
-            </div>
-          </div>
-        )}
-
         <Loading isLoading={isLoading} className={styles[loadClass]}>
-          {isCenterPage ? (
-            <div className={styles.centerPage}>{children}</div>
+          {isCenterPage || hasBack ? (
```

Comment:
Move this logic to a method is more clear, maybe called `renderTopNav`

Reply:
There is refactor in other branch